src/pages/workspace/accounting/xero/XeroTaxesConfigurationPage.tsx

```diff
@@ -1,59 +1,42 @@
 import React from 'react';
-import {View} from 'react-native';
-import HeaderWithBackButton from '@components/HeaderWithBackButton';
-import OfflineWithFeedback from '@components/OfflineWithFeedback';
-import ScreenWrapper from '@components/ScreenWrapper';
-import ScrollView from '@components/ScrollView';
-import Switch from '@components/Switch';
-import Text from '@components/Text';
+import ConnectionLayout from '@components/ConnectionLayout';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Connections from '@libs/actions/connections';
-import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import * as ErrorUtils from '@libs/ErrorUtils';
 import type {WithPolicyProps} from '@pages/workspace/withPolicy';
 import withPolicyConnections from '@pages/workspace/withPolicyConnections';
-import variables from '@styles/variables';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as Policy from '@userActions/Policy';
 import CONST from '@src/CONST';
 
 function XeroTaxesConfigurationPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
-    const {importTaxRates, pendingFields} = policy?.connections?.xero?.config ?? {};
+    const xeroConfig = policy?.connections?.xero?.config;
```

Comment:
```suggestion
    const xeroConfig = policy?.connections?.xero?.config ?? {};
```
And you don't have to add empty object in each place later